docs/development-environment/design/notifications.rst

```diff
@@ -0,0 +1,83 @@
+Notifications
+=============
+
+Notifications are a critical component of user experience (UX) in digital products, serving as a bridge between the system and the user. They should enhance, not detract from, the user experience, assisting users in achieving their goals and providing immediate, relevant feedback. This guide synthesizes best practices from the Carbon Design System and industry standards to help developers create effective and user-friendly notifications.
```

Comment:
Hm, it was a mistake to leave this part there
Without more instructions, devs could have issues to understand since Carbon uses React while my implementation is based on Bootstrap
Carbon could be useful for the dev creating/changing components appearance, but it will probably be unusable for the ones who just want to create the code without thinking about how it's going to look in details...

Reply:
I mean, perhaps the dev experience here would be like "Just put a div with the classes alert and the other you want then you're good to go"